roles/tofu/tasks/remote_prepare.yml

```
@@ -18,9 +18,11 @@
   ansible.builtin.file:
     path: '{{ cifmw_tofu_dir_path }}'
     state: directory
+    mode: "0755"
 
 - name: "Copy Local Tufo Project Directory To Remote Host '{{ ansible_host }}'"
   ansible.builtin.copy:
+    mode: "0644"
```

As `cifmw_tofu_project_path` is an unvalidated user input the user could provide an input that doesn't end with `/` meaning the path would be recursively copied. [1] 

We should also add `directory_mode: "0755"` to cover for this to be explicit with our copied directory modes as well.[2]

```suggestion
    mode: "0644"
    directory_mode: "0755"
```

[1] https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html#parameter-src
[2] https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html#parameter-directory_mode